lib/bios/menus.robot

```diff
@@ -412,6 +413,48 @@ Reset To Defaults Tianocore
     Read From Terminal Until    ignore.
     Write Bare Into Terminal    y
 
+    # For platforms using PiKVM there is a need to turn on Serial Redirection otherwise most of tests might break.
```

Comment:
I do not get it. If serial is not enabled, how this code can work, as it reads from serial? This seems pointless, we simply have to modify the test binary via DCU prior running tests, which is already documented.

Unless I'm missing something here, then please correct me.

Reply:
Ok, I think I do get it on the second review. We want to enable serial after we reset settings to default, to ensure the serial remains enabled, even if it has just been disabled by resetting.

Reply:
This is part of `Reset To Defaults`, and default is disabled. DCU doesn't change the default, only the current setting.

Reply:
If this is the case, it is not necessarily tied to PiKVM directly in any way.

This is simply needed for devices where serial is disabled by default. Which may or may not be related to the fact that they do have PiKVM connected as well.

What is more, it should not hurt to make sure this options is selected even on devices where serial is enabled after resetting to defaults.

Reply:
Fortunately resetting options to default does not turn off serial redirection until DUT is reset. So the option can be turned on before so.
My understanding was that atm only PiKVM enabled platforms use serial redirection. However it is a good idea to differentiate where this option should be enabled by some other way.

Reply:
@JanPrusinowski please link the issue here